### ISSUE TITLE
Some faucets not work

### ISSUE THREAD

**Issue by didshc** [ExploreBTC/BitcoinGrind#3]
I'm having issues with some faucets. They just do not open. I've seen this happen in other rotators and I know it's not exactly a script problem, but in some rotators when this happens they simply load a page saying "this faucet needs to be opened in another window", like a link. Does anyone know how to do this?

**Comment by goldkey0070** [comment]
Yes, it's called sand something.  Its a code to prevent  I framing a website.  You can look up blocking of I framing a website to get better under standing . There is a way to have a anti I frame block bur I ferret how. Just look up in Google how to use I frame block code or anti I frame Blick code etc.

**Comment by goldkey0070** [comment]
make a list instead for those faucets to open in new window like this.

`<span><strong><a style="color: #33cccc;" href="http://bitcoinfaucetexchange.com" target="_blank">Dimensional BTC</a></strong></span>`
<span><strong><a style="color: #33cccc;" href="http://bitcoinfaucetexchange.com" target="_blank">Dimensional BTC</a></strong></span>
